northcoders/frontend/games-project/src/components/ReviewCard.jsx

```diff
@@ -0,0 +1,28 @@
+import { Link } from 'react-router-dom'
+
+export default function ReviewCard({review_id, owner, title, category}){
+    return (
+        <>
+        <br /> 
+        
+        
+        <section id="itemCard"> 
+        <Link to={`/reviews/${review_id}`}>
```

Comment:
Does this all need to be a link, it could be confusing for a user to click the category and reach the individual article.